src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/MatchType.cs

```diff
@@ -5,8 +5,8 @@ namespace Microsoft.AspNetCore.Rewrite.Internal.IISUrlRewrite
 {
     public enum MatchType
     {
-        Pattern,
-        IsFile,
-        IsDirectory
+        pattern,
```

Comment:
I think not. These should be reverted as they are never exposed to end user code. These are internal implementation details of our URLRewrite engine. So the casing should just be what it was before and we should parse case insensitive from the file format (which is the only thing that matters).